src/Raven.Server/Smuggler/Migration/MigrationConfiguration.cs

```diff
@@ -35,6 +35,8 @@ public abstract class MigrationConfigurationBase
 
         public int BuildVersion { get; set; }
 
+        public string ProductVersion { get; set; }
```

Comment:
IS it required?
![image](https://user-images.githubusercontent.com/1250431/42944400-d88bb2fe-8b65-11e8-9482-35426a239193.png)